test/unit/isolated/require-page.test.ts

```diff
@@ -70,9 +70,11 @@ describe('getPagePath', () => {
 
 describe('requirePage', () => {
   it('Should not find page /index when using /', async () => {
-    await expect(() => requirePage('/', distDir, false)).toThrow(
-      'Cannot find module for page: /'
-    )
+    try {
+      await requirePage('/', distDir, false)
+    } catch (err) {
+      expect(err.message).toBe('Cannot find module for page: /')
+    }
```

Comment:
This seems wrong, as if `requirePage` does not throw, then this test will still pass. The previous `.toThrow` should be used here.